app/scripts/background.js

```diff
@@ -1155,6 +1165,18 @@ export function setupController(
         });
       }
 
+      ///: BEGIN:ONLY_INCLUDE_IF(build-experimental)
+      if (processName === ENVIRONMENT_TYPE_SIDEPANEL) {
+        sidePanelIsOpen = true;
+        finished(portStream, () => {
+          sidePanelIsOpen = false;
```

Comment:
what if there are multiple side panels open?

Reply:
Chrome's Side Panel API only allows ONE side panel to be open per browser window. So, I am not sure if there is a potential issue here. Do you see any issues that I should look at?